src/GT.Core/Services/Impl/GTListingService.cs

```diff
@@ -191,17 +192,17 @@ public async Task<List<ListingOverviewDTO>> GetAsync(IListingFilterModel? filter
 					filter.Location == null
 					|| (e.Location != null
 						&& e.Location.Name != null
-						&& string.Equals(filter.Location, e.Location.Name)))
+						&& EF.Functions.Like(e.Location.Name, filter.Location)))
 
 				.Where(e =>
 					filter.SalaryMin == null
-					|| (e.SalaryMax != null
-						&& e.SalaryMax >= filter.SalaryMin))
+					|| (e.SalaryMin != null
+						&& e.SalaryMin >= filter.SalaryMin))
 
 				.Where(e =>
 					filter.SalaryMax == null
-					|| (e.SalaryMin != null
-						&& e.SalaryMin <= filter.SalaryMax))
+					|| (e.SalaryMax != null
+						&& e.SalaryMax <= filter.SalaryMax))
```

Comment:
Salary max should be 
`filter.SalaryMax >= e.SalaryMin`

Reply:
Shouldn't it compare the maximum salary values for entities and only return those entities for which the maximum salary value is less than or equal to the maximum salary given by the filter? i.e. providing 45000 as the maximum salary value in the filter should only return listing entities where the maximum salary for that listing is less than 45000.

It's possible that I've misunderstood the purpose of this part of the filter, maybe it would be a good idea to have a short meeting about it after DSTUM on 06/04-22?

Reply:
Agree with Anton, 
Trying a visual representation of a slider below, where we as of now set internal boundaries of a horisontal scale:
<--minSalary -- <-filterMin----------filterMax-> ----- maxSalary-->.

Reply:
As an applicant you want to see all job listings which could offer you a salary which falls within an acceptable salary range. 
Say that an applicant states it has a range of 10-20, meaning that the applicant could accept 10 as the lowest salary offer. 
A job listing with the range of 5-11 could still be of interest to the applicant as the applicant could receive an offer from the company which falls within the requested range (an offer of 10 and 11).

The filter should return all listings where the filter and the listing ranges overlap. The current implementation excludes listings from showing up even with listings of which an acceptable offer could be met.

Reply:
We have now discussed this feature and have decided on implementing it in the way that Rasmus and Kristoffer suggested. This PR will be merged with the intention of fixing it in a future PR so that the other functionality in this PR can be implemented in the main branch and used by the other developers ASAP.

Reply:
Referencing branch for future reference when implementing the agreed upon range filtering. #96.